build.gradle.kts

```diff
@@ -66,41 +66,53 @@ subprojects {
     if (!this.name.contains("sample") && this.name != "sentry-test-support") {
         apply<DistributionPlugin>()
 
+        val sep = File.separator
+
         configure<DistributionContainer> {
             this.getByName("main").contents {
                 // non android modules
-                from("build/libs")
-                from("build/publications/maven")
+                from("build${sep}libs")
+                from("build${sep}publications${sep}maven")
                 // android modules
-                from("build/outputs/aar")
-                from("build/publications/release")
+                from("build${sep}outputs${sep}aar")
+                from("build${sep}publications${sep}release")
             }
         }
+
+        val rootDir = this.project.rootProject.projectDir
+        val distDir = File("${rootDir}${sep}distributions")
+
+        // create dir if it does not exist
+        distDir.mkdirs()
+
         tasks.named("distZip").configure {
             this.dependsOn("publishToMavenLocal")
             this.doLast {
-                val distributionFilePath = "${this.project.buildDir}/distributions/${this.project.name}-${this.project.version}.zip"
+                val distributionFilePath = "${this.project.buildDir}${sep}distributions${sep}${this.project.name}-${this.project.version}.zip"
                 val file = File(distributionFilePath)
                 if (!file.exists()) throw IllegalStateException("Distribution file: $distributionFilePath does not exist")
                 if (file.length() == 0L) throw IllegalStateException("Distribution file: $distributionFilePath is empty")
+
+                val newFile = File("${distDir}${sep}${file.name}")
+                file.copyTo(newFile, overwrite = true)
+
+                UnzipUtils.unzip(newFile, distDir.path)
+                newFile.delete()
```

Comment:
ideally we don't do that, but rather download artifacts from GH artifacts and extract locally